src/PositionService.Data/PositionRepository.cs

```diff
@@ -74,6 +74,14 @@ public async Task<List<DbPosition>> GetAsync(IGetPositionsRequest request)
       return await dbPosition.ToListAsync();
     }
 
+    public async Task<List<DbPosition>> GetPositionsAsync(List<Guid> positionsIds)
```

Comment:
you in positionRepository, do not use the word Position in Method name

Reply:
fixed